packages/web-platform/web-worker-rpc/etc/web-worker-rpc.api.md

```diff
@@ -0,0 +1,128 @@
+## API Report File for "@lynx-js/web-worker-rpc"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// Warning: (ae-forgotten-export) The symbol "RpcEndpointAsyncVoid" needs to be exported by the entry point index.d.ts
```

Comment:
We need to resolve these warnings to ensure high-quality auto-generated documentation.

But maybe in the next PRs.